app/components/appMainModule/examplePage1Module/testService1.js

```diff
@@ -57,7 +57,7 @@
         .then(getPageDataComplete)
         .catch(getPageDataFailed);
 
-      function getPageDataComplete(data, status, headers, config) {
+      function getPageDataComplete(data) {
```

Comment:
I'm not sure I follow. If it's used on line 85, should I leave it there? I'll leave it for now and we can always delete it later. The point was mostly to make the linter happy and get the tests to pass.